### DISCUSSION TITLE
Let me know if you would like to take ownership of hill/tldr-chrome

### DISCUSSION THREAD

**Discussion by hill** [piraces/tldr-extension-browser#6] [General]
Hi @piraces.

I've archived hill/tldr-chrome as I really don't have time to maintain. This fork is awesome!

I've pointed the README to this fork, however let me know if you want to take over the original repo as people still seem to end up there 😆 .

**Comment by piraces** [comment]
Hi @hill , thank you for that!

I think for now that it's enough and clear for the people.
Otherwise, I will let you know 👍
